src/commands/license.ts

```diff
@@ -0,0 +1,8 @@
+import { Command } from '@oclif/command'
+import { checkLicense } from '../common'
```

Comment:
Do we have it here? I can't see it in this PR.

Reply:
in common.js we have import like import { findExistingDefaultLicenseTemplate, checkLicense } from './features/license'

and next it is exported, so we have it.